common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -464,4 +464,14 @@ trait FeatureSwitches {
     exposeClientSide = false,
   )
 
+  val EmailSignupRecaptcha = Switch(
```

Comment:
Out of curiosity, what are the intentions for this switch, is it to help us more confidently migrate to using recaptcha in email sign ups? Or is it intended to exist forever in case we need to quickly/easily disable recaptcha on signups? I guess the thing that made me think about this is 'sellByDate = never'

Reply:
Good point! Perhaps my understanding of `sellByDate` is incorrect, but I intended to keep this until we either decide to:

* integrate captcha permanently into the codebase & remove the switch
* turn it off in the face of bugs 
* remove captcha entirely

I don't think we want a situation where the switch expires and toggles `off`, without selecting one of these outcomes first.

Reply:
Thanks for explaining - the sell by date definitely makes sense then :)